lib/matplotlib/backends/backend_qt5.py

```diff
@@ -17,10 +18,19 @@
 from matplotlib.backends.qt_editor._formsubplottool import UiSubplotTool
 from . import qt_compat
 from .qt_compat import (
-    QtCore, QtGui, QtWidgets, __version__, QT_API,
-    _devicePixelRatioF, _isdeleted, _setDevicePixelRatioF,
+    QtCore,
+    QtGui,
+    QtWidgets,
+    _getSaveFileName,
+    __version__,
+    QT_API,
+    _devicePixelRatioF,
+    _isdeleted,
+    _setDevicePixelRatioF,
```

Comment:
Please revert this change, i.e., `_getSaveFileName` is not used, and the restyling is unnecessary churn.